jobs/build/pre-release/Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def prerelease = load("../release/release.groovy")
+    def buildlib = prerelease.buildlib
+    def commonlib = prerelease.commonlib
+    def quay_url = "quay.io/openshift-release-dev/ocp-release-nightly"
+
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'FROM_RELEASE_TAG',
+                        description: 'Build tag to pull from (i.e. 4.1.0-0.nightly-2019-04-22-005054)',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'NAME',
+                        description: 'Release name (i.e. 4.1.0-rc0)',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'DESCRIPTION',
```

Comment:
unlikely we'll have any use for description or previous in this job, but if it makes code reuse easier, that's OK.